### ISSUE TITLE
Build fails: error: cannot initialize a parameter of type 'npy_intp *' (aka 'int *') with an rvalue of type 'std::__1::vector<long, std::__1::allocator<long> >::value_type *' (aka 'long *')

### ISSUE THREAD

**Issue by yurivict** [TRIQS/cpp2py#40]
```
/wrkdirs/usr/ports/devel/cpp2py/work/cpp2py-10ab97d2e5e56656227a74118943de02e2265292/c++/cpp2py/numpy_proxy.cpp:35:87: error: cannot initialize a parameter of type 'npy_intp *' (aka 'int *') with an rvalue of type 'std::__1::vector<long, std::__1::allocator<long> >::value_type *' (aka 'long *')
       PyArray_NewFromDescr(&PyArray_Type, PyArray_DescrFromType(element_type), rank, extents.data(), strides.data(), data, flags, NULL);
                                                                                      ^~~~~~~~~~~~~~
1 error generated.
```

Log: http://beefy10.nyi.freebsd.org/data/113i386-default/549079/logs/cpp2py-1.5.20200812.log

This seems to a i386-only failure.

**Comment by Wentzell** [comment]
This issue should be fixed in 44d5238

**Comment by yurivict** [comment]
Thanks!
